tests/integration-tests/tests/networking/test_security_groups.py

```diff
@@ -106,19 +122,30 @@ def test_overwrite_sg(region, scheduler, custom_security_group, pcluster_config_
         _assert_security_group_rules(
             ec2_client,
             mount_target_security_group_id,
-            [mount_target_security_group_id, custom_security_group_id],
+            [mount_target_security_group_id, custom_security_group_ids[0]],
         )
-
     if scheduler == "slurm":
         logging.info("Checking SSH connection between cluster nodes before cluster update")
         _check_connections_between_head_node_and_compute_nodes(cluster)
+
+        # Update the Security Groups of HN to check if the HN instance is replaced.
```

Comment:
let's use HeadNode rather than HN. AFAIK the HN acronym is not used in other places and Head Node is more self-explicative.